halyard/halScripts/setupMonitoringContent.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
This should probably be deleted as it will get added to the setupMonitoring.sh script and that already has a shebang